lib/ansible/galaxy/api.py

```diff
@@ -342,7 +362,7 @@ def _call_galaxy(self, url, args=None, headers=None, method=None, auth_required=
 
                 return res
 
-            elif not url_info.query:
+            elif not ('page' in query or 'offset' in query):
```

Comment:
I must've explicitly mentioned that the var could be reused here too:
```suggestion
            elif not is_paginated_url:
```